client/api_payment.go

```diff
@@ -61,6 +81,18 @@ func (c *Client) Deposit(ctx context.Context, toAddress string, amount math.Int,
 }
 
 // Withdraw withdraws BNB from a stream account.
+//
```

Comment:
Can we add more details about the withdaw? It may have some restriction, and withdraw too much will cause force settle?

Reply:
Added more information.